src/probabilistic.jl

```diff
@@ -126,7 +126,7 @@ function ProbabilisticMetadata(time_series::Probabilistic)
     )
 end
 
-convert_data(data::AbstractDict{Dates.DateTime, T}) where {T <: Matrix{<:Real}} =
```

Comment:
Why isn't this written as:

```suggestion
convert_data(data::AbstractDict{Dates.DateTime, Matrix{T}}) where T <: Real =
```

Reply:
In Julia when a container is defined inside of a container it needs to be defined as abstract.

Reply:
I think the answer to Dheepak's question "tunnel vision" (I wrote this).  His suggestion is good and we should make the change.